src/ziggurat/middleware/json.clj

```diff
@@ -0,0 +1,35 @@
+(ns ziggurat.middleware.json
+  "This namespace defines middleware methods for parsing JSON strings.
+   Please see [Ziggurat Middleware](https://github.com/gojek/ziggurat#middleware-in-ziggurat) for more details.
+  "
+  (:require [cheshire.core :refer :all]
+            [sentry-clj.async :as sentry]
+            [ziggurat.sentry :refer [sentry-reporter]]
+            [ziggurat.metrics :as metrics]))
+
+(defn- deserialize-json
+  [message topic-entity-name key-fn]
+  (try
+    (parse-string message key-fn)
+    (catch Exception e
```

Comment:
Why generic exception?

Reply:
@kartik7153  Because, Cheshire library does not say which exceptions can be thrown.